ast2vec/repo2/base.py

```diff
@@ -251,18 +281,34 @@ def process_repo(self, url_or_path, output) -> bool:
         :return: True if the operation was successful; otherwise, False.
         """
         repo2 = self.WORKER_CLASS(**self._args)
+        if os.path.exists(output):
+            if repo2.override_existing:
+                self._log.info("Model %s already exists, but will be override. If you want to "
+                               "skip existing models use --no_override flag", output)
+            else:
+                self._log.info("Model %s already exists, skipping.", output)
+                return True
         try:
             result = repo2.convert_repository(url_or_path)
             for proto in ("https://", "http://", "git://", "ssh://"):
                 if url_or_path.startswith(proto):
                     url_or_path = url_or_path.replace(proto, "")
             model = self.WORKER_CLASS.MODEL_CLASS()
+            self._log.info("Construct %s model...", url_or_path)
+            start = time()
             model.construct(**self.result_to_model_kwargs(result, url_or_path))
+            self._log.info("Construct %s model is done. time: %f", url_or_path, time() - start)
```

Comment:
All timings must be activated under DEBUG and the code executed if the logger.isEnabledFor